pom.xml

```diff
@@ -1611,6 +1615,7 @@
         <hbase.version>${hbase-2.4.0.runtime.version}</hbase.version>
         <hbase.thirdparty.version>3.4.1</hbase.thirdparty.version>
         <zookeeper.version>3.5.7</zookeeper.version>
+        <tephra.hbase.compat.version>2.4</tephra.hbase.compat.version>
```

Comment:
Tephra module doesn't have any compat issues b/ HBase 2.4.0 and 2.4.1 (the way Phoenix has) right?

Reply:
Yes, that's my understanding.